alot/settings/manager.py

```diff
@@ -311,8 +311,7 @@ def colourpick(triple):
     def get_hook(self, key):
         """return hook (`callable`) identified by `key`"""
         if self.hooks:
-            if key in self.hooks.__dict__:
-                return self.hooks.__dict__[key]
+            return getattr(self.hooks, key, None)
```

Comment:
No, with `getattr` if you don't provide a fallback then an `AttributeError` is raised.
https://docs.python.org/2/library/functions.html#getattr